docs/test_specification.yaml

```diff
@@ -0,0 +1,218 @@
+definitions: 
+  hello1: 
```

Comment:
maybe just name these samples respectively:
hello_signed
hello_signed_upgrade_confirmed
hello_signed_upgrade_bad_key
hello_signed_upgrade_wrong_key
etc.

Reply:
The unsigned image should probably by also checked as update option.

Reply:
Apart from signing, the versioning should also be checked.